Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -411,15 +423,29 @@ public void StartConnectLoginWithSteamAppTicket(EOSManager.OnConnectLoginCallbac
 #if DISABLESTEAMWORKS
             onLoginCallback?.Invoke(new Epic.OnlineServices.Connect.LoginCallbackInfo() { ResultCode = Epic.OnlineServices.Result.UnexpectedError });
 #else
+            // If this method is called and the login fails, we need to clean out the cached encryptedAppTicket.
+            EOSManager.OnConnectLoginCallback cacheInvalidationOnFailureCallback = (Epic.OnlineServices.Connect.LoginCallbackInfo callbackInfo) => 
+            {
+                // First, determine if this is a failure. If it is, then clear out the cache.
+                if (callbackInfo.ResultCode != Epic.OnlineServices.Result.Success)
+                {
+                    Debug.LogWarning($"SteamManager will clear the cached {nameof(this.encryptedAppTicket)} after failed Connect login attempt using this Steam App Ticket.");
```

Comment:
This has been adjusted, and adjusted a bit further.

Now the callback is only modified in the event where RequestAppTicket succeeds, rather than the modification running in the case where RequestAppTicket fails to get a token.

The log has been demoted to informational. I think your insight here that being a 'Warning' makes it sound like we're demanding action from the user, but really we just want to inform them of something.

No action should be required on the user's part. Does the new comment suit our needs?